eq-author-api/src/validation/schemas/entities.json

```diff
@@ -1,31 +1,52 @@
 {
   "$id": "http://example.com/schemas/entities.json",
   "definitions": {
-    "page": {
+    "section": {
       "type": "object",
       "properties": {
         "id": { "type": "string" },
         "title": {
-          "$ref": "definitions.json#/definitions/populatedString"
+          "type": "string"
+        },
+        "pages": {
+          "type": "array",
+          "items": {
+            "$ref": "entities.json#/definitions/page"
+          }
         }
       },
-      "required": ["id", "title"]
+      "required": ["id", "title", "pages"]
     },
-    "section": {
+    "page": {
       "type": "object",
       "properties": {
         "id": { "type": "string" },
         "title": {
-          "type": "string"
+          "$ref": "definitions.json#/definitions/populatedString"
         },
-        "pages": {
+        "answers": {
           "type": "array",
           "items": {
-            "$ref": "entities.json#/definitions/page"
+            "$ref": "entities.json#/definitions/answer"
           }
         }
       },
-      "required": ["id", "title", "pages"]
+      "required": ["id", "title"]
+    },
+    "answer": {
+      "type": "object",
+      "properties": {
+        "id": { "type": "string" }
+      },
+      "if": { "properties": { "type": { "enum": ["TextField", "Number"] } } },
```

Comment:
There are Currency, Percentage and TextArea as well

Reply:
The trello ticket specified Numeric & Text fields only:
[https://trello.com/c/pu89dqzO/1102-validator-basic-answer-4](https://trello.com/c/pu89dqzO/1102-validator-basic-answer-4)

Reply:
Currency and Percentage are numeric fields. And we have two text fields - text field and text area.

Reply:
Now changed to include TextField, Curreny & Percentage